docs/docsite/rst/guide_migration.rst

```diff
@@ -28,4 +31,331 @@ Module ``win_iis_website``
 
 Migrated to :ref:`microsoft.iis.website <ansible_collections.microsoft.iis.website_module>`.
 
-To complete.
+community.windows.win_iis_website has been split into two modules:
+  * ansible.microsoft.iis.website.
+  * ansible.microsoft.iis.website_info.
+
+Custom site Parameters from string ($parameters) removed from the module
+win_iis_webbinding was consolidated into microsoft.iis.website
```

Comment:
This should be under it's own section for migrating the `community.windows.win_iis_webbinding` instead.